### PR TITLE
implementation and migration added

### PR DESCRIPTION
**Pull request by ulrokx**

### CODE REVIEW

File: api/src/bs/Launch/updateAvgDiffPercent.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const { connectDb } = require("../../knex/knex");
+
+// should be called after screenshot is finished processing
+module.exports = async ({ launchId }) => {
+  const knex = await connectDb();
+  const avg = await knex("Screenshot")
+    .avg("diffPercentage")
+    .where({ launchId });
+  if (!avg) {
```
Comment (aduyng) [inline api/src/bs/Launch/updateAvgDiffPercent.js:9]: this will fail in case of no screenshot, the value could be 0

**Review by aduyng — CHANGES_REQUESTED** [review]
comments

File: api/src/bs/Launch/updateAvgDiffPercent.js (hunk 1 above)
Comment (ulrokx) [reply api/src/bs/Launch/updateAvgDiffPercent.js:9]: okay - fixed it - if there are no screenshots with that launch id it returns null

File: api/src/cli/updateAvgDiffPercent.js [hunk 2]
```diff
@@ -1,3 +1,3 @@
 const updateAvgDiffPercent = require("../bs/Launch/updateAvgDiffPercent");
 
-module.exports = () => updateAvgDiffPercent();
+module.exports = (...argv) => updateAvgDiffPercent(...argv);
```
Comment (aduyng) [inline api/src/cli/updateAvgDiffPercent.js]: why is the `...` needed?

**Review by aduyng — CHANGES_REQUESTED** [review]
comments

File: api/src/cli/updateAvgDiffPercent.js (hunk 2 above)
Comment (ulrokx) [reply api/src/cli/updateAvgDiffPercent.js]: the ... dots are ... not ... needed

**Review by aduyng — APPROVED** [review]

**Pull request merged**